src/scripts.js

```diff
@@ -178,16 +208,23 @@ function displayRecipes(recipes) {
           </div>
        </section>
      `;
-      // if (currentUser.favorites.includes(recipe)) {
-      //   // selectFavoriteButton();
-      //   // e.target.value = 'favorited';
-      //   // e.target.classList.add('favorited-state');
-      // } else {
-      //   deselectFavoriteButton();
-      // }
   });
+
   createCardEventListeners();
   createButtonEventListeners();
+
+  recipeCards = document.querySelectorAll('.recipe-card-js');
```

Comment:
logic that styles the cards correctly-> are they already selected/not and apply appropriate styling and value